_rules/html-page-title-descriptive-c4a8a4.md

```diff
@@ -37,18 +37,15 @@ htmlHintIgnore:
 
 ## Applicability
 
-This rule applies to the first [HTML][] `title` element that
-
-- is a [descendant](https://dom.spec.whatwg.org/#concept-tree-descendant) of the `html` element of a [web page](https://www.w3.org/TR/WCAG21/#dfn-web-page-s), and
-- contains [children](https://dom.spec.whatwg.org/#concept-tree-child) that are [text nodes](https://dom.spec.whatwg.org/#text) that are not only [whitespace](#whitespace).
+This rule applies to the [document title][] of each [html web page][] that contains [text nodes](https://dom.spec.whatwg.org/#text) that are not only [whitespace](#whitespace).
 
 ## Expectation
 
 The target element describes the topic or purpose of the overall content of the [document](https://dom.spec.whatwg.org/#concept-document).
 
 ## Assumptions
 
-- A prerequisite for this rule is that Technique H25: Providing a title using the title element is used for living up to 2.4.2 Page Titled
+There are currently no assumptions.
```

Comment:
I think the Assumption still exists, but maybe in a slightly different form. See #1924 (an email written as HTML is an "html web page", the HTML document might have an empty `title` element but the email's Subject is actually used as the "title" (in WCAG sense), so it wouldn't fail the SC).
OTOH, we might let that as is and tackle that in another PR when solving #1924.

Reply:
> I think the Assumption still exists, but maybe in a slightly different form. See #1924 (an email written as HTML is an "html web page", the HTML document might have an empty `title` element but the email's Subject is actually used as the "title" (in WCAG sense), so it wouldn't fail the SC). OTOH, we might let that as is and tackle that in another PR when solving #1924.

+1 to keeping the assumption until we resolve that issue.